core-build/gulp-core-build/src/tasks/CleanTask.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+/* tslint:disable:no-trailing-whitespace whitespace */ /* Remove this when GCB-TS is published and upgraded */
```

Comment:
There is no whitespace - TSLint is confused with two versions of TS in the same repo. I debugged into it and couldn't find a resolution, so I'm disabling the rules temporarily until I upgrade GCB in this repo.

---
In reply to: [129395072](https://github.com/Microsoft/web-build-tools/pull/276#discussion_r129395072) [](ancestors = 129395072)